src/ck-core/ckcallback.h

```diff
@@ -534,6 +534,27 @@ class CkCallback {
                   CkAbort("Tried to set a refnum on a callback not directed at an entry method");
                 }
         }
+
+int epIndex(void) const {
+  switch (type) {
+    case isendChare:
+    case sendChare:
+      return d.chare.ep;
+    case isendGroup:
+    case sendGroup:
+    case isendNodeGroup:
+    case sendNodeGroup:
+    case bcastNodeGroup:
+    case bcastGroup:
+      return d.group.ep;
+    case isendArray:
+    case sendArray:
+    case bcastArray:
+      return d.array.ep;
+    default:
+      return -1;
```

Comment:
Will this function actually return -1 for any real case, or should it just abort like the above method?

Reply:
This will return `-1` for all non-EP targeting callbacks (e.g., futures) on `ckcallback.C:333`.